modules/ROOT/pages/functions/string.adoc

```diff
@@ -148,6 +148,163 @@ RETURN ltrim('   hello')
 
 ======
 
+
+
+[[functions-normalize]]
+== normalize()
+
+`normalize()` returns the given `STRING` normalized using the `NFC` normalization form.
+
+The `normalize()` function is useful for converting `STRING` values into comparable forms.
+When comparing two `STRING` values, it is their Unicode codepoints that are compared.
+In Unicode, a codepoint for a character that looks the same may actually be represented by two, or more, different codepoints.
+For example, the character `<` can be represented as `\uFE64` or `\u003C`. Visually, the character may look the same,
+but if compared, Cypher will return false as `\uFE64` does not equal `\u003C`. Using the `normalize()` function one can
+normalize the codepoint `\uFE64` to `\u003C`, creating a single codepoint representation, allowing them to be successfully compared.
+
+*Syntax:*
+
+[source, syntax, role="noheader"]
+----
+normalize(input)
+----
+
+*Returns:*
+
+|===
+
+| `STRING`
+
+|===
+
+*Arguments:*
+
+[options="header"]
+|===
+| Name | Description
+
+| `input`
+| An expression that returns a `STRING`.
+
+|===
+
+*Considerations:*
+
+|===
+
+| `normalize(null)` returns `null`.
+
+|===
+
+
+.+normalize()+
+======
+
+.Query
+[source, cypher, indent=0]
+----
+RETURN normalize('\u212B') = '\u00C5' AS result
+----
+
+.Result
+[role="queryresult",options="header,footer",cols="1*<m"]
+|===
+
+| +result+
+| +true+
+1+d|Rows: 1
+
+|===
+
+======
+
+
+[[functions-normalize-with-normal-form]]
+== normalize(), with specified normal form
+
+`normalize()` returns the given `STRING` normalized using the specified normalization form.
+The normalization form can be of type `NFC`, `NFD`, `NFKC` or `NFKD`.
+
+There are two main types of normalization forms. One is based on the concept of canonical equivalence, and the other is based on compatibility.
+
+The two forms `NFC` (default) and `NFD` are forms of canonical equivalence. This means that codepoints which represent the same abstract character will
+be normalized to the same codepoint. The same abstract character means that the character has the same visual appearance and behavior.
+The difference between `NFC` and `NFD` is that `NFC` form will always give the *composed* canonical form, one where combined codes are replaced with the single representation, if possible.
+Whereas, `NFD` gives the *decomposed* form, this is the opposite of composed, converting combined codepoints into the split form if possible.
+
+The two forms `NFKC` and `NFKD` are forms of compatibility normalization. All canonically equivalent sequences are compatible, but not all compatible sequences are canonical.
+This means that a character normalized in `NFC` or `NFD` should also be normalized in `NFKC` and `NFKD`, and other characters that may have a slightly different visual appearance,
+but are considered close enough in appearance to be compatibly equivalent.
```

Comment:
Im not sure I understand the last part of this section ("other characters that may have a slightly different visual appearance,
but are considered close enough in appearance to be compatibly equivalent.") 
What other characters are we referring to here?

```suggestion
* *Compatability normalization*: `NFKC` and `NFKD` are forms of compatibility normalization.
All canonically equivalent sequences are compatible, but not all compatible sequences are canonical.
This means that a character normalized in `NFC` or `NFD` should also be normalized in `NFKC` and `NFKD`.
Other characters with only slight differences in appearance should be compatibly equivalent.
```

Reply:
That is what the example below tries to illustrate, but basically lets say the ﬀ char. This is one Unicode (\uFB00), this char is visually distinguishable from ff (ﬀ != ff : in other fonts this is more obvious, but you may notice if you try to highlight it the first one is one char, and the second is 2 f chars).

Now these have different meanings, two f's are just two letters and can be used to spell a word e.g fluff, but if I spelt fluff as fluﬀ then the length of that word is now 4 instead of 5, because only unicode chars are written. This also is not the two letters ff, same as in Swedish, the Å is not A. So they are not canonically equivalent, as they do not have the same behaviour when used (e.g Å is not pronounced A). But if I am an English speaker, searching a database for your name I would probably spell Åklundh as Aklundh, and would expect the database to still return your name, this is how normalization using compatibility works, whilst I have changed the meaning of your name, in a search way, they are still "compatible" with each other. This type of normalization should not be used to render information, e.g Slack shouldn't normalize your name away, even if the search does :)